src/config-schema.ts

```diff
@@ -46,12 +52,11 @@ export const configSchema = {
   },
   showSchemaSaveWarning: {
     _type: Type.Boolean,
-    _default: true,
     _description: 'Whether to show a warning about possibly losing data in the forms dashboard',
+    _default: true,
   },
   dataTypeToRenderingMap: {
     _description: 'A map used to match concept datatypes to rendering types',
-    _type: Type.Object,
```

Comment:
Why are we getting rid of the type here?

Reply:
I'm guessing this comment is outdated?

Reply:
Yeah now that I'm reading this config schema as you've rewritten it, it is definitely a shame that the upper-level descriptions aren't working, because it would indeed be helpful here.